packages/opencensus-web-core/src/trace/model/tracer.ts

```diff
@@ -108,4 +131,9 @@ export class Tracer extends TracerBase implements webTypes.Tracer {
 
   /** Binds trace context to NodeJS event emitter. No-op for opencensus-web. */
   wrapEmitter(emitter: webTypes.NodeJsEventEmitter) {}
+
+  isZonePresent(): boolean {
+    //tslint:disable:no-any
```

Comment:
Rather than using `any`, could you create a new interface to cast `window` to?

For example:
```
interface WindowWithZone {
   Zone?: Function
}
```